### DISCUSSION TITLE
Some Problems with the Example Sequence Diagram

### DISCUSSION THREAD

**Discussion by MucahitErdoganUnlu** [bounswe/bounswe2024group11#165] [General]
This is the example that we covered in the PS.
![image](https://github.com/bounswe/bounswe2024group11/assets/94074126/b2a4d2f2-7b53-449a-9538-c31789bef600)
This is the class diagram:
![image](https://github.com/bounswe/bounswe2024group11/assets/94074126/c32e5530-24cc-4043-a7c9-edd7f1099d83)

As can be seen on the class diagram, createNote belongs to the User object, and Note method belongs to the Note object. However, in the above solution, it seems that createNote method belongs to UserInterface object (which does not exist in the class diagram) and Note method belongs to the Database. 

This is how I think the solution should be:
![image](https://github.com/bounswe/bounswe2024group11/assets/94074126/8707bdfe-a13c-4566-a93d-aa3ed85fa92e)

What do you think? @furkanakkurt1335 
@tenacke @hks1444

**Comment by furkan7258** [comment]
Hey Mücahit! Thank you for the question. 

'User' and 'UserInterface' are not objects but actors. The interface seems to be also doing the system's job in the example from the PS.

**Reply by MucahitErdoganUnlu** [reply]
Thanks for the answer! My solution is also correct I guess.
